### PR TITLE
Blast size enhancer item

### PR DESCRIPTION
**Pull request by tigleym**
Adds a blast size enhancer to the item pool. This work also modifies `spawn_blast_explosion` to take the blast entity's `Transform` component since the modification from the item is applied to it in [BlasterComponent.fire](https://github.com/amethyst/space_shooter_rs/blob/cdsupina-dev/src/components/weapons.rs#L131-L135).

### CODE REVIEW

**Comment by tigleym** [comment]
For #58

File: src/systems/item.rs [hunk 1]
```diff
@@ -167,6 +167,10 @@ impl<'s> System<'s> for ItemSystem {
                             spaceship.set_health(spaceship.max_health);
                         }
 
+                        if item.stat_effects.contains_key("blast_size_multiplier") {
+                            blaster.size_multiplier *= item.stat_effects["blast_size_multiplier"];
```
Comment (tigleym) [inline src/systems/item.rs:171]: This should assign the value of item.stat_effects["blast_size_multiplier"] to the blaster's size_multiplier, not multiply it.

Comment (cdsupina) [reply src/systems/item.rs:171]: I think that we should change this to a `+=` and then change the item's stat effect value to 4.0. This way if we choose to add more items in the future that modify blasts' size we won't have multipliers of multipliers which could quickly get out of control.

**Review by cdsupina — APPROVED** [review]
I left a comment about how the blast multiplier is applied.

I also think that items with multiple animation frames should have their own row for now, so I would reccommend moving all of the frames to a row directly below the existing items.

Other than that it looks good! Merge when ready!

**Review by cdsupina — APPROVED** [review]

**Pull request merged**